src/components/02-components/data-table/data-table-experiences.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import {
+  Typography,
+  useTheme,
+} from '@mui/material';
+import OpenInBrowserIcon from '@mui/icons-material/OpenInBrowser';
+import EmailIcon from '@mui/icons-material/Email';
+import {
+  DataGrid,
+  GridColDef,
+  GridToolbarContainer,
+  GridToolbarExport,
+  GridToolbarColumnsButton,
+  GridToolbarFilterButton,
+  GridToolbarDensitySelector,
+  GridRowId,
+  GridActionsCellItem,
+} from '@mui/x-data-grid';
+
+export type Row = {
+  id: GridRowId;
+  col1: string;
+  col2: string;
+  col3: string;
+  col4: string;
+  col5: string;
+  col6: string;
+};
+
+interface DataTableProps {
+  initialRows: Row[];
+}
+
+export default function DataTableExperiences({ initialRows }: DataTableProps) {
+  const theme = useTheme();
+
+  function emailUser(col3: string) {
+    return () => {
+      window.location.href = 'mailto:' + col3;
+    };
+  }
+
+  function viewExperience(id: GridRowId) {
+    return () => {
+      window.location.href = '/organization/experiences/' + id;
+    };
+  }
+
+  const columns: GridColDef<Row>[] = [
+    { field: 'col1', headerName: 'First Name', width: 150 },
+    { field: 'col2', headerName: 'Last Name', width: 150 },
+    { field: 'col3', headerName: 'Email', width: 150 },
+    { field: 'col4', headerName: 'Campus', width: 150 },
+    { field: 'col5', headerName: 'Offering', width: 150 },
+    { field: 'col6', headerName: 'Status', width: 150 },
+    {
+      field: 'col7',
+      headerName: 'Actions',
+      width: 150,
+      type: 'actions',
+      disableExport: true,
+      getActions: (params) => [
+        <GridActionsCellItem
+          icon={<EmailIcon />}
+          label="EMAIL USER"
+          onClick={emailUser(params.row.col3)}
+          showInMenu
+        />,
+        <GridActionsCellItem
+          icon={<OpenInBrowserIcon />}
+          label="VIEW DETAILS"
```

Comment:
We avoid capitalized words due screen readers, if any text should be capitalized that needs to be done using css
```suggestion
          label="View details"
```